OpenKeychain/src/main/java/org/sufficientlysecure/keychain/remote/ui/SelectAuthKeyIdActivity.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2015 Dominik Schürmann <dominik@dominikschuermann.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.sufficientlysecure.keychain.remote.ui;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+import org.openintents.openpgp.util.OpenPgpApi;
+import org.openintents.ssh.SSHAgentApiConstants;
+import org.sufficientlysecure.keychain.Constants;
+import org.sufficientlysecure.keychain.R;
+import org.sufficientlysecure.keychain.operations.results.OperationResult;
+import org.sufficientlysecure.keychain.ui.base.BaseActivity;
+import org.sufficientlysecure.keychain.util.Log;
+
+public class SelectAuthKeyIdActivity extends BaseActivity {
```

Comment:
please rewrite this based off RemoteDeduplicateActivity. looks way nicer :)

Reply:
Actually, looking over that activity I noticed it's far easier to do it without the use of fragments. I'll look into that soonish. Put this on hold for now.

Reply:
Already done :). I did a simple copy&paste RemoteDeduplicateActivity, but it doesn't overlay properly over the calling app.

Reply:
heh, alright then.

as for the overlay issues: set the of the activity to transparent in the manifest file :)